dll/win32/kernel32/kernel32_vista/SetFileInformationByHandle.c

```diff
@@ -0,0 +1,89 @@
+/*
+
+// TODO
+
+ * COPYRIGHT:       See COPYING in the top level directory
+ * PROJECT:         BtrFS FSD for ReactOS
+ * FILE:            dll/shellext/shellbtrfs/reactos.cpp
+ * PURPOSE:         ReactOS glue for Win8.1
+ * PROGRAMMERS:     Pierre Schweitzer <pierre@reactos.org>
```

Comment:
Please strip the FILE part. And follow the new style for those headers reactos.org/wiki/Coding_Style

Reply:
Yeah: this header is marked as `TODO`.
I'll update it after my TODO question about WINE import is answered.